docs/Commands.md

```diff
@@ -229,7 +229,7 @@ FT.CREATE idx ON JSON SCHEMA $.title AS title TEXT $.categories AS categories TA
         is to be split into individual tags. The default is `,`. The value
         must be a single character.
 
-
+    * **CASESENSITIVE**
```

Comment:
Doing JSON array and case sensitive in the same PR?

Reply:
will split